#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise a NameError exception with a message.

    Args:
        message: The error message to include in the exception
    """
    raise NameError(message)
